### DISCUSSION TITLE
Failed helm install

### DISCUSSION THREAD

**Discussion by Juuuub** [requarks/wiki#6088] [Error / Bug Report]
Seems that helm chart is not found or available? Worked a few days ago when I tried it.
`helm fetch requarks/wiki`

`Error: failed to fetch https://charts.js.wiki/charts/wiki-2.2.18.tgz : 404 Not Found`

**Comment by Juuuub** [comment]
For now I have specified version 2.2.0 manually.
`helm fetch requarks/wiki --version 2.2.0`

**Comment by skyreacher** [comment]
I'm also experiencing problems with the helm repo today. I randomly get the same `404 Not Found` error. But when trying several times, it happens to work at some point.
I've had this problem just today, yesterday it worked without issues. I suggest you simply retry a few times...
